src/components/filter/types/FilterDateRange.vue

```diff
@@ -46,6 +71,11 @@ export default {
     FilterBoilerplate
   },
   extends: FilterAbstract,
+  data() {
+    return {
+      placeholderMask: ''
```

Comment:
Any reason why `placeholderMask` is not a computed property?

Reply:
yes it's because it's based on a ref and from what I understood refs can't be "computed" that's why I reassign it on locale change: https://github.com/ICIJ/datashare-client/blob/d2c3747ace755bb6e7e550b4101370f887911e9f/src/components/filter/types/FilterDateRange.vue#L129

Reply:
Makes sense! Thanks